GitFyle.Core.Api/Services/Foundations/Configurations/ConfigurationService.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using GitFyle.Core.Api.Brokers.DateTimes;
+using GitFyle.Core.Api.Brokers.Loggings;
+using GitFyle.Core.Api.Brokers.Storages;
+using GitFyle.Core.Api.Models.Foundations.Configurations;
+
+namespace GitFyle.Core.Api.Services.Foundations.Configurations
+{
+    internal partial class ConfigurationService : IConfigurationService
+    {
+        private readonly IStorageBroker storageBroker;
+        private readonly ILoggingBroker loggingBroker;
+        private readonly IDateTimeBroker dateTimeBroker;
+
+        public ConfigurationService(
+            IStorageBroker storageBroker,
+            ILoggingBroker loggingBroker,
+            IDateTimeBroker dateTimeBroker)
+        {
+            this.storageBroker = storageBroker;
+            this.loggingBroker = loggingBroker;
+            this.dateTimeBroker = dateTimeBroker;
+        }
+
+        public ValueTask<Configuration> AddConfigurationAsync(Configuration configuration) =>
+            TryCatch(async () =>
+            {
+                await ValidateConfigurationOnAdd(configuration);
+                return await this.storageBroker.InsertConfigurationAsync(configuration);
```

Comment:
```suggestion
                await ValidateConfigurationOnAdd(configuration);

                return await this.storageBroker.InsertConfigurationAsync(configuration);
```
Always a line break before return statement.